### PR TITLE
handle an incorrect logic

### PR DESCRIPTION
**Pull request by eirture**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:

If the key not found in podStates, the ps is nil, and we will get a nil pointer panic

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by yangjunmyfm192085** [comment]
needs CLA

**Comment by ajieliu** [comment]
I signed it

**Comment by ajieliu** [comment]
Please review again, thanks!

/cc wzshiming yangjunmyfm192085

File: pkg/scheduler/internal/cache/cache.go [hunk 1]
```diff
@@ -734,6 +734,7 @@ func (cache *schedulerCache) cleanupAssumedPods(now time.Time) {
 		ps, ok := cache.podStates[key]
 		if !ok {
 			klog.Fatal("Key found in assumed set but not in podStates. Potentially a logical error.")
+			continue
```
Comment (adtac) [inline pkg/scheduler/internal/cache/cache.go:737]: calls to klog.Fatal will terminate the program, so this line will never be executed

Comment (ajieliu) [reply pkg/scheduler/internal/cache/cache.go:737]: get, I will close this PR. Thanks for review

**Pull request closed without merging**